tests/manual/monkey/build.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-This build requires javafx.home property, pointing to JavaFX SDK (20+) directory, e.g.:
+This build requires javafx.home property, pointing to JavaFX SDK (21+) directory, e.g.:
   ant -Djavafx.home=<DIR>
 -->
 <project default="build-all" basedir=".">
```

Comment:
If you add the following property as a default for `javafx.home`, then you can build it using just "ant" without having to specify it on the command line:

```
    <property name="javafx.home" value="../../../build/sdk" />
````